src/command_modules/azure-cli-interactive/azclishell/telemetry.py

```diff
@@ -36,12 +36,12 @@ class Telemetry(TelemetryClient):
     @_user_agrees_to_telemetry
     def track_ssg(self, gesture, cmd):
         """ track shell specific gestures """
-        self.track_event('Shell Specific Gesture', {gesture: scrub(cmd)})
+        self.track_event('az/interactive/gesture/{}'.format(gesture), {gesture: scrub(cmd)})
```

Comment:
No. Only three parts. You don't need to have the gesture name in the event name. Make it a property of the event body.